cdc/sinkv2/eventsink/txn/txn_sink_test.go

```diff
@@ -16,26 +16,30 @@ package txn
 import (
 	"bytes"
 	"context"
+	"errors"
 	"sort"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/golang/mock/gomock"
+	"github.com/pingcap/log"
 	"github.com/pingcap/tidb/parser/mysql"
 	"github.com/pingcap/tiflow/cdc/model"
 	"github.com/pingcap/tiflow/cdc/sinkv2/eventsink"
-	"github.com/pingcap/tiflow/pkg/notify"
+	mock_txn "github.com/pingcap/tiflow/cdc/sinkv2/eventsink/txn/mock"
 	"github.com/stretchr/testify/require"
 )
 
 type blackhole struct {
```

Comment:
Maybe we can rename it mockBackend because we have a blackhole sink in sinkv2. Perhaps it will help to reduce the confusion.